hardware/Sterbox.cpp

```diff
@@ -13,7 +13,7 @@
 
 extern CSQLHelper m_sql;
 
-#define STERBOX_POLL_INTERVAL 10
+#define STERBOX_POLL_INTERVAL 8

```

Comment:
Okey thanks for the explanation
I can not merge this, because it has conflicts, i might have changed a line to solve a compiler warning